format/private/format.sh

```diff
@@ -13,6 +13,9 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
   { echo>&2 "ERROR: runfiles.bash initializer cannot find $f. An executable rule may have forgotten to expose it in the runfiles, or the binary may require RUNFILES_DIR to be set."; exit 1; }; f=; set -e
 # --- end runfiles.bash initialization v3 ---
 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source "$SCRIPT_DIR/ls-files.sh"
```

Comment:
source "$SCRIPT_DIR/private/ls-files.sh"

Reply:
This script is already in `/private` so that doesn't feel like the right fix... looking

Reply:
There's not a good answer for the runfiles lookup that would be needed. Went back to a single format.sh script to avoid inventing something complicated.